app/src/main/java/com/youssef/fixit/Auth/LoginFragment/ISharedPreferenceInterface.java

```diff
@@ -0,0 +1,13 @@
+package com.youssef.fixit.Auth.LoginFragment;
+
+import android.content.SharedPreferences;
+
+import com.google.gson.annotations.SerializedName;
+
+interface ISharedPreferenceInterface {
```

Comment:
Naming convention wise, `I` Here stands for Interface so you don't need to have both I and Interface you can rename it `ISharedPreference` and it will be clear that it's an interface.